services/api/src/api/config.py

```diff
@@ -111,7 +111,10 @@ class EndpointConfig:
             "/parquet-and-dataset-info": {"dataset": ["/parquet-and-dataset-info"]},
             "/parquet": {"dataset": ["/parquet"]},
             "/dataset-info": {"dataset": ["/dataset-info"]},
-            "/sizes": {"dataset": ["/sizes"]},
+            "/sizes": {
+                "dataset": ["/dataset-size"],
```

Comment:
Here the same, we could at first use the old "/splits" and then if not available or with error point to "/dataset-size":
```suggestion
                "dataset": ["/splits", "/dataset-size"],
```
Once we validate that it works point only to                 "dataset": ["/dataset-size"],